.github/workflows/tests.yml

```diff
@@ -0,0 +1,53 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Hopefully unnecessary, but if we're already here?

```
on:
   push:
     branches:
       - main
       - '[0-9].[0-9]'
```
WDYT?

Reply:
Okay 👌